ibm_watson/visual_recognition_v3.py

```diff
@@ -165,18 +157,20 @@ def classify(self,
                                               images_file_content_type or
                                               'application/octet-stream')))
         if url:
-            url = str(url)
             form_data.append(('url', (None, url, 'text/plain')))
         if threshold:
-            threshold = str(threshold)
-            form_data.append(('threshold', (None, threshold, 'text/plain')))
+            form_data.append(
+                ('threshold', (None, str(threshold), 'text/plain')))
         if owners:
-            owners = self._convert_list(owners)
-            form_data.append(('owners', (None, owners, 'text/plain')))
+            for item in owners:
+                form_data.append(('owners', (None, item, 'text/plain')))
```

Comment:
This should be sending as comma separated string

Reply:
I agree, java does this `multipartBuilder.addFormDataPart("owners", RequestUtils.join(classifyOptions.owners(), ","))`
probably something like this might do the trick
`form_data.append(('owners', (None, ",".join(owners), 'text/plain')))`

Reply:
yeah should not be updated